src/Solver/Simulator.cpp

```diff
@@ -128,6 +134,9 @@ void seissol::Simulator::simulate(seissol::SeisSol& seissolInstance) {
     if (upcomingTime < m_currentTime + l_timeTolerance) {
       logError() << "Simulator did not advance in time from" << m_currentTime << "to" << upcomingTime;
     }
+    if (m_abort) {
```

Comment:
You could also add a message here like `logInfo(seissol::MPI::mpi.rank()) << "Aborting simulation.";` or the likes.